roles/ceph-mon/tasks/set_rgw_pool_pg_num.yml

```diff
@@ -0,0 +1,9 @@
+# Uncomment below lines to create pools for the rgw 
```

Comment:
These are variables in a dict so they should go into: `roles/ceph-mon/defaults/main.yml`